cognite/neat/_graph/extractors/_classic_cdf/_classic.py

```diff
@@ -389,7 +389,7 @@ def _extract_asset_parent_data_sets(self):
             for chunk in self._chunk(
                 list(self._asset_parent_uri_by_id.keys()), description="Extracting asset parent data sets"
             ):
-                assets = self._client.assets.retrieve_multiple(id=list(chunk), ignore_unknown_ids=True)
+                assets = self._client.assets.retrieve_multiple(ids=list(chunk), ignore_unknown_ids=True)
```

Comment:
This like it should have been caught by type checker, no?

Reply:
@larshagencognite yes, nice catch. I had a quick lock in the mypy rules for the repo, currently does not have all rules to catch this kind of error. 

I will talk to the team for us to enforce this, it will probably come in different PR.

Reply:
@larshagencognite FYI -> https://github.com/cognitedata/neat/pull/1055 This PR adds the proper mypy rules to the repo.